plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/TestDeltaLakeAdlsConnectorSmokeTest.java

```diff
@@ -89,30 +88,26 @@ DockerizedDataLake createDockerizedDataLake()
         hadoopCoreSiteXmlTempFile.toFile().deleteOnExit();
         Files.write(hadoopCoreSiteXmlTempFile, abfsSpecificCoreSiteXmlContent.getBytes(UTF_8));
 
-        return new DockerizedDataLake(
-                getHadoopBaseImage(),
-                ImmutableMap.of(),
-                ImmutableMap.of(hadoopCoreSiteXmlTempFile.normalize().toAbsolutePath().toString(), "/etc/hadoop/conf/core-site.xml"));
+        HiveMinioDataLake hiveMinioDataLake = new HiveMinioDataLake(
+                bucketName,
+                ImmutableMap.of(hadoopCoreSiteXmlTempFile.normalize().toAbsolutePath().toString(), "/etc/hadoop/conf/core-site.xml"),
+                "ghcr.io/trinodb/testing/hdp3.1-hive");
```

Comment:
Pls avoid using magic strings for the docker image - Let's introduce a constant for this purpose.